gems/actionpack/CVE-2008-7248.yml

```diff
@@ -1,5 +1,5 @@
 ---
-gem: rails
+gem: actionpack
```

Comment:
Will probably also need `framework: rails`, even though `bundler-audit` doesn't use it, but other downstream consumers might need it.